.env.example

```diff
@@ -62,4 +62,7 @@ AWS_DEFAULT_REGION=us-east-1
 AWS_BUCKET=
```

Comment:
LGTM.

No changes or comments needed on .env.example as it's an example file and not part of production code.